frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -345,6 +346,21 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [MCG_FLAG],
     },
   },
+  // When RGW is available without MCG
+  {
+    type: 'Page/Resource/List',
+    properties: {
+      model: models.NooBaaObjectBucketModel,
+      loader: () =>
+        import(
+          './components/object-bucket-page/object-bucket' /* webpackChunkName: "object-bucket-page" */
+        ).then((m) => m.ObjectBucketsPage),
+    },
+    flags: {
+      required: [RGW_FLAG],
+      disallowed: [MCG_FLAG],
+    },
```

Comment:
We are only setting RGW Flag if MCG Flag is set: https://github.com/openshift/console/blob/5e6250cb5313cb783b88e3088d45e275a7615de6/frontend/packages/ceph-storage-plugin/src/plugin.ts#L113

Reply:
oops!